example/fedora/minimal-40-aarch64.yaml

```diff
@@ -1,7 +1,14 @@
 otk.version: 1
 
 otk.define:
-  version: 40
-  architecture: "aarch64"
+  otk.op.join:
```

Comment:
Is this change driven by the current implementation or is this the yaml we want? I'm asking because it does not feel like an improvement and I think at this stage of the project we should aim for the best possible yaml and then think about the implementation (and being able to have a toplevel with `otk.define` and an include that also `otk.defines` stuff without the need for an explicit `otk.op.join` seems nice.

Reply:
This is driven by the current implementation and what is possible in it. And yes; the 'way it is' (in `main`) is nicer :)